TestPlaybooks/playbook-IPInfo_PY_Test.yml

```diff
@@ -0,0 +1,194 @@
+id: IPInfoTest
```

Comment:
this should be the same as the original tests file once you override the js integration.
validate that and remove this file